.flowconfig

```diff
@@ -9,6 +9,9 @@ dist
 [lints]
 
 [options]
+# Cheap workaround to make flow typechecking work across packages in the monorepo
```

Comment:
I think I had to do this because we no longer use flow-mono-cli. @janpaul123 Do you remember why it was removed?

Reply:
Yeah it didn't seem to do anything — Flow seemed to keep passing just fine without it. Any idea why we needed it now particularly in this PR? Anyway, this seems like a simpler solution than their whole symlink business. 😄

Reply:
> Any idea why we needed it now particularly in this PR? 

I think this is the first time `// @flow` was added to stuff in the docs folder. 😬 

> Anyway, this seems like a simpler solution than their whole symlink business.

Yeah, and my solution is probably incomplete / wrong in some edge cases too 😅

Reply:
> Yeah, and my solution is probably incomplete / wrong in some edge cases too 😅

Haha I dunno! It's basically the same as we have in our private webviz repo and that seems to be working just fine!

Reply:
it's working for me too